### PR TITLE
Basic example

### PR DESCRIPTION
**Pull request by junji-genesys**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,76 @@
...
+      b. Type `cd {name of your PureCloud Embeddable Framework project}`.<br />
+      c. Type `sudo node server.js`.
+2. In your browser, go to https://localhost/example.html.
+3. Log in to the client.
+
+You can now interact with the example page and explore the PureCloud Embeddable Framework APIs in real-time. For example, to see how click-to-dial works, click the click-to-dial link on the example page.
+
+# Additional Information
+
+This content is [licensed](https://github.com/MyPureCloud/purecloud-for-salesforce-einstein-example/blob/master/LICENSE) under the MIT license.
```
Comment (kristenoliphant) [inline README.md]: Just an FYI this link should be to the license in the repo.

File: src/framework.js [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+window.Framework = {
+    config: {
+        name:"ExampleGitHubApp",
+        clientIds: {
+            'mypurecloud.com': '',
+            'mypurecloud.ie': '',
+            'mypurecloud.com.au': '',
+            'mypurecloud.jp': '',
+            'mypurecloud.de': ''
```
Comment (kristenoliphant) [inline src/framework.js]: Do we want to add US west?

File: README.md (hunk 1 above)
Comment (junji-genesys) [reply README.md]: Good catch! I'll update the link.

File: src/framework.js (hunk 2 above)
Comment (junji-genesys) [reply src/framework.js]: Good for now until usw2 goes GA.

**Pull request merged**